help/en/package/jmri/jmrit/webThrottle/CamerawebThrottle.shtml

```diff
@@ -0,0 +1,102 @@
+
+      <p>Usage: &lt;your JMRI server IP&gt;:&lt;port&gt;/web/webThrottle.html</p>
+
+      <p>This application shows feedback and controls not only locomotives, but also turnouts,
+      routes, lights and other devices through JMRI panels.</p>
+
+      <p><strong>webThrottle</strong> is completely resizable and it adjusts automatically to the
+      size and orientation of the device screen.</p>
+
+      <p>It runs on the most recent browsers on computers, smartphones and tablets.<br>
+      Your device and browser must support HTML5 and websockets.<br>
+      Tested on Safari, Firefox and Google Chrome running on Windows, MAC, iPad/iPhone and Android
+      - it should run on Internet Explorer 10.<br>
+      <strong>There is nothing to install on your device !</strong></p>
+
+      <p>To start using JMRI through a web browser, you just need to activate the included
+      <strong>Web Server</strong>. You may found additional information <a href=
+      "/help/en/html/web/index.shtml">here</a>.</p>
+
+      <p>To get help, please click '?' at each page top right.<br>
+      You will find important and useful information there.<br>
+      There is specific Help for each page type (Roster, Loco, Panel, ...).</p>
+
+      <h3>Screen samples...</h3>
+
+      <p><img src="images/JMRI_webThrottle.png" height="300" alt="The default screen">
+      </p>
+
+      <p><img src="images/JMRI_webThrottle-(loco).png" height="300" alt="A loco throttle">
+      </p>
+
+      <p><img src="images/JMRI_webThrottle-(panel).png" height="300" alt="An interactive panel">
+      </p>
+
+      <p><img src="images/JMRI_webThrottle-(iframes).png" height="300" alt=
+      "Multi throttle screens in iframes">
+      </p>
+
+      <p>The following setup will be handled in two sections, one for local internal streaming and control, and the next for streaming to remote engineers on the internet. Local streaming is good for testing or if you end up with too many engineers for a room, but not for the layout itself. Internal setup is useful to do before, as much of the setup applies to remote engineers on the internet.</p>
+      <p>You must also have every engine you want to run with the CabCam added to JMRI's roster. You can get as fancy or as spartan as you want with this, you can just add <strong>NMRA standard CV definitions</strong> decoders to your roster, the important bit is the DCC address is in the roster so it shows up in WebThrottle.</p>
+      <p><strong>Internal Setup</strong>First you are going to need to get the streaming URL for your CabCam which can be obtained from the <a href="https://siliconvalleylines.com/2022/08/11/cabcam-goes-uv4l/" target="_blank" rel="noreferrer noopener">CabCam setup instructions.</a> Setting the Memory Variables. Open Tools&gt;Tables&gt;Memory Variables.</p>
+      
+      <p><img src="images/JMRI_webThrottle-(tools).png" alt="JMRI Tools Menu" /></figure></p>
```

Comment:
I don't see any start tag, only end tag, for `<figure>`.

Reply:
Thanks, removing that one now. I it has been a bit since I have done manual modification of HTML. I removed all </figure> since it shouldn't be needed.